src/App.js

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+// import ChatEntry from './components/ChatEntry';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  // brain
+  const [ChatData, setChatData] = useState(chatMessages);
+  const [likeCount, setLikeCount] = useState(0);
+  const updateLike = (id, like) => {
+    // console.log(id, ChatData);
+    // console.log(like);
+    // console.log(id.liked);
+
+    const newMessage = ChatData.find((entry) => entry.id === id);
+    // console.log('newMessage');
+    // console.log(newMessage);
+    // console.log(like);
+    newMessage.liked = like;
+    setChatData([...ChatData]);
```

Comment:
The `newMessage` is never put back into `chatData`, so the hearts in the ChatEntry components never change color, which is why one of the Wave 3 tests doesn't pass.